### ISSUE TITLE
Task scripts:watch compiles twice

### ISSUE THREAD

**Issue by ereze** [Swiip/generator-gulp-angular#851]
The task "scripts:watch" is dependent on [scripts] which turns out to compile twice the code. Was that done intentionally?

Regads

**Comment by Swiip** [comment]
Yes, I know. To be honest, I still doesn't have well understood why but it fails if you remove it. We had issues (#459) when it was not there.

But I agree, it's done twice and should be faster. Any hint would be appreciated.
